src/label/_label.scss

```diff
@@ -15,9 +15,38 @@
     @include govuk-text-colour;
 
     display: block;
+
+    margin-bottom: $govuk-spacing-scale-1;
+  }
+
+  // Modifiers that make labels look more like their equivalent headings
+
+  .govuk-label--xl {
```

Comment:
If we used the core heading classes would we need to do this?

Reply:
i prefer that it's separate.

Reply:
The intention behind this is to decouple the idea of _what's a heading_ with what's important on the page - reframing the problem so that `heading-xl` is 'a heading which is extra large', and `label--xl` is 'a label that is extra large' – you can have an medium label which is not a heading, and you could (theoretically) have a label with no additional styling that is a heading. Using the core heading classes muddies the semantics again.

There are some additional benefits in that we don't actually want any of the margins that the core heading classes give us, so at the minute we have to override them - whereas this allows us to define them separately.

Reply:
Think it's important to think about how the user views these, both `label--xl` and `heading--xl` is the same thing visually.

I'm mostly concerned about:

- how we keep the two in sync
- impact on file size
- another interface to learn

That said, if it's not easy to use the core classes, this seems a reasonable approach.

Other benefit is that decoupling them makes them easier to delete.

Reply:
I should have prefixed my comment with that it wasnt blocking.

Just wanted to understand that using the core classes was not possible, since I think this is very complex to understand.

Reply:
It's totally possible, but I don't think it's actually desirable in this case, for the reasons outlined above.